roles/installer/templates/deployments/deployment.yaml.j2

```
@@ -371,6 +371,10 @@ spec:
 {% if ee_extra_env -%}
             {{ ee_extra_env | indent(width=12, first=True) }}
 {% endif %}
+# Optional sidecars to be added to the deployment:
+{% if extra_sidecars %}
```

The expression must be closing with `-%` instead of `%` to remove redundant whitespaces that break YAML syntax.

```suggestion
{% if extra_sidecars -%}
```